core/commands/active.go

```diff
@@ -22,7 +22,7 @@ Lists running and recently run commands.
 		res.SetOutput(req.InvocContext().ReqLog.Report())
 	},
 	Options: []cmds.Option{
-		cmds.BoolOption("v", "verbose", "print more verbose output"),
+		cmds.BoolOption("v", "verbose", "print more verbose output").Default(false),
```

Comment:
Is it necessary to add `Default(false)` to every boolean option? I think it can be inferred that the default value is false.